### DISCUSSION TITLE
Can't get Node.js example running

### DISCUSSION THREAD

**Discussion by twiesing** [openobserve/openobserve#4768] [Q&A] [answered]
Hi,

I wanted to monitor an application with OpenObserve. I took https://openobserve.ai/docs/ingestion/traces/nodejs as an example.

Unfortunately, I get an error when sending the logs/traces (tried with the cloud and self-hosted variant):

`{"stack":"OTLPExporterError: Not Found\n    at IncomingMessage.<anonymous> (/Users/thueske/Development/Spielwiese/sample-tracing-nodejs-javascript/sample-tracing-nodejs-javascript/node_modules/@opentelemetry/otlp-exporter-base/build/src/platform/node/util.js:103:39)\n    at IncomingMessage.emit (node:events:531:35)\n    at endReadableNT (node:internal/streams/readable:1696:12)\n    at process.processTicksAndRejections (node:internal/process/task_queues:82:21)","message":"Not Found","name":"OTLPExporterError","data":"404 page not found","code":"404"}`

I have entered the data from the backend 1:1:

```
const sdk = new opentelemetry.NodeSDK({
  // traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
  traceExporter: new opentelemetry.tracing.ConsoleSpanExporter(),
  traceExporter: new OTLPTraceExporter({
    url: "https://api.openobserve.ai/api/tobias_organization_49221_ASDWKMVCIWEKMC",
    headers: {
      Authorization: "Basic dG9SUPERSECUR3=",
    },
  }),
```

Any ideas?

**Comment by Manazsharma** [comment]
Hi, 
This looks like a misconfiguration error, you should verify the auth header and URL again. If you need help sending logs from a node.js application consider taking a look at this blog  https://openobserve.ai/blog/sending-pino-logs-to-openobserve.

**Reply by twiesing** [reply]
The header and URL are copied from the openobserve backend... Tested multiple times.

**Reply by Manazsharma** [reply, marked as answer]
try replacing the URL with this:
`https://api.openobserve.ai/api/tobias_organization_49221_ASDWKMVCIWEKMC/v1/traces`
